### PR TITLE
server/shadow: Fix black screen with win10 mstsc

### PR DESCRIPTION
**Pull request by realjiangms**
Request full screen update on RDPGFX_CAPS_ADVERTISE_PDU. Win10 client seems to clean the screen after gfx channel opened. If there happens to be no screen update from server, we will get black screen in mstsc client.

### CODE REVIEW

**Comment by akallabeth** [comment]
@realjiangms Pull looks good so far, could you rebase it to current master?

**Comment by realjiangms** [comment]
Yes I'll do it later.

File: server/shadow/shadow_client.c [hunk 1]
```diff
@@ -615,6 +615,70 @@ static UINT shadow_client_rdpgfx_qoe_frame_acknowledge(RdpgfxServerContext*
...
+	}
+	for (index = 0; index < capsAdvertise->capsSetCount; index++)
+	{
+		if (pdu.capsSet->version == RDPGFX_CAPVERSION_8)
+		{
+			if (settings)
+			{
+				flags = pdu.capsSet->flags;
+				settings->GfxThinClient = (flags & RDPGFX_CAPS_FLAG_THINCLIENT);
+				settings->GfxSmallCache = (flags & RDPGFX_CAPS_FLAG_SMALL_CACHE);
```
Comment (akallabeth) [inline server/shadow/shadow_client.c]: @realjiangms Shouldn't the GfxH264 be set to the default value here?

Comment (realjiangms) [inline server/shadow/shadow_client.c]: GfxH264 is set in next line.

**Comment by realjiangms** [comment]
Can we get this one Merged?

**Pull request merged**